src/main/java/seedu/revised/ui/Ui.java

```diff
@@ -101,56 +101,62 @@ public static void printBye() {
                 + "____________________________________________________________");
     }
 
-    public static void printError() {
-        System.out.println("____________________________________________________________\n"
+    public static String printFailedParseError() {
```

Comment:
Personally I don't think it's a good idea to make Ui to return string the exception message in this way if no cuztomization is to be done to the message. Constants would be better in this case.